### PR TITLE
Categories/rules

### PR DESCRIPTION
**Pull request by chanchal16**
- Created categories and rules page.
- quizzes listed by categories.
- rules to play

https://user-images.githubusercontent.com/56486295/160459933-54b1a912-a3e2-440b-9249-46bb3c6160c2.mp4

### CODE REVIEW

**Review by sandeepkumardev-zz — APPROVED** [review]
everything well organised

**Comment by Anasooya2601** [comment]
Good to go👍🏻

**Review by Anasooya2601 — APPROVED** [review]

File: src/styles/categories.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+/* overlay */
+.overlay{
+    background-color: transparent;
+    top: 10rem;
+}
+.overlay-text{
+    background-color: rgba(0, 0, 0, 0.3);
+    color: white;
```
Comment (poojavishnoi) [inline src/styles/categories.css:24]: use var for color, in this line and above this also.

**Review by poojavishnoi — COMMENTED** [review]
Code is clean.Good to merge, just left a suggestion

**Pull request merged**